src/components/Table/Table/__stories__/table.stories.module.scss

```diff
@@ -0,0 +1,15 @@
+.scrollTable {
+  height: 220px;
+  width: 800px;
+}
+
+.virtualizedScrollTable {
+  display: flex;
+  flex-direction: column;
+  height: 250px;
+  flex: 1;
+}
+
+.iconRule {
+  width: 95%;
+}
```

Comment:
since classnames aren't available in the story (and aren't copy-pastable), they should be inlined.
I'm also not sure why `.virtualizedScrollTable` and `. scrollTable` should be set manually and not built-in in the table